process/interface.go

```diff
@@ -31,6 +31,16 @@ type TxTypeHandler interface {
 	ComputeTransactionType(tx data.TransactionHandler) (TransactionType, error)
 }
 
+// TxHandlerProcessValidator can determine if a provided transaction handler is valid or not from the process point of view
+type TxHandlerProcessValidator interface {
+	CheckTxHandlerValid(txHandler data.TransactionHandler) bool
+}
+
+// HeaderHandlerProcessValidator can determine if a provided header handler is valid or not from the process point of view
+type HeaderHandlerProcessValidator interface {
```

Comment:
interface name too long: HeaderValidator should be enough. Same for transactions as well: TransactionValidator. Should rename function to IsTransactionValid and IsHeaderValid

Reply:
HeaderValidator and TxValidator